webpack-stats-production.json

```diff
@@ -0,0 +1 @@
+{"status":"done","chunks":{"main":[{"name":"main.js","path":"/Users/audiosutras/caktus_projects/philly-hip/hip/static/js/bundles/main.js"},{"name":"main.js.map","path":"/Users/audiosutras/caktus_projects/philly-hip/hip/static/js/bundles/main.js.map"}]}}
```

Comment:
I assume that this file is generated but I'm guessing that we don't want your local directory in here.

Reply:
done